pages/blog/integrating-mailchimp-with-your-nodejs-app.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: "Integrating MailChimp with your Node.js App"
+date: "2021-06-01"
+author: "Yashovardhan Agrawal"
+pullQuote: "A step by step guide of integrating your MailChimp Audiences with your Node.js app"
+image: "integrating-mailchimp-with-your-nodejs-app/mc4.png"
+tags: [connections, engineering]
+---
+
+# Integrating MailChimp with your Node.js App
+
+If you're reading this blog right now, the chances are that you're finding your way to add a MailChimp integration into your Node.js application. Don't worry; we have you covered! This is a step by step guide that will help you build a sample Node.js backend for your webpage and help you integrate it with the MailChimp API.
+This tutorial will discuss all the different things you need to look for within MailChimp with references to their excellent API documentation. We'll also talk about an alternative method using Grouparoo, which will do all the heavy lifting for you!
```

Comment:
Not sure if you were trying to start a new paragraph here or not, but you'll want two newlines to create a new paragraph.

Reply:
added

Reply:
oh yeah, didn't notice it earlier. Thanks for pointing it out! Removing the heading.